.github/workflows/daily-build.yml

```diff
@@ -17,12 +17,12 @@ jobs:
         with:
           distribution: 'temurin'
           java-version: 17.0.7
-
+      
       - name: Set environment variable
         if: github.event.action == 'check_connector_for_breaking_changes'
         run: |
           echo "BUILD_USING_DOCKER=-PbuildUsingDocker=nightly" >> $GITHUB_ENV
-          echo "GRADLE_SKIP_TASKS=-x :nats-compiler-plugin-tests:test -x :nats-ballerina-tests:build -x :nats-ballerina-tests:test" >> $GITHUB_ENV
+          echo "GRADLE_SKIP_TASKS=-x :nats-compiler-plugin-tests:test"  >> $GITHUB_ENV
```

Comment:
Do we need these values to be set as env.s here? Can be directly applied in the next step as it seems. (Also, any specific reason to disable the compiler plugin tests?)